cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/ApiController.java

```diff
@@ -254,6 +256,94 @@ private static void writeTask(JsonGenerator json, String name, CassandraFramewor
             json.writeNullField(name);
     }
 
+    @GET
+    @Path("/live-nodes/{forTool}")
+    public Response liveEndpoints(@PathParam("forTool") String forTool) {
+        return liveEndpoints(forTool, 1);
+    }
+
+    @GET
+    @Path("/live-nodes/{forTool}/{nodeCount}")
+    public Response liveEndpoints(@PathParam("forTool") String forTool, @PathParam("nodeCount") Integer nodeCount) {
```

Comment:
I would prefer each of these scenarios be split out into individual methods.

I think we should switch to using query params for limit rather than path element. All URL methods could specify a `limit` query param `@QueryParam("limit") @DefaultValue("3")` where we can set a default limit if one isn't specified.

In terms of URLs I think we should tweak to be (with target mime type)
* `/live-nodes` -> `application/json`
* `/live-nodes/tool/cqlsh` -> `text/x-apache-cassandra-cqlsh`
* `/live-nodes/tool/stress` -> `text/x-apache-cassandra-stress`
* `/live-nodes/tool/nodetool` -> `text/x-apache-cassandra-nodetool`

I'm having a hard time imagining a case where `ascii` would be useful. If you don't have a use case in mind please remove it.